### PR TITLE
775-fe-migrate-backbutton-to-storybook

### LINKED ISSUE #775: [FE] Migrate BackButton component to storybook

### PR DESCRIPTION
**Pull request by REALcuba**
Closes #775

### CODE REVIEW

File: packages/ui/.storybook/main.ts [hunk 1]
```diff
@@ -7,6 +7,8 @@ const config: StorybookConfig = {
     "@storybook/addon-essentials",
     "@storybook/addon-onboarding",
     "@storybook/addon-interactions",
+    '@storybook/addon-actions',
+    'storybook-addon-react-router-v6'
```
Comment (kevinmamaqi) [inline packages/ui/.storybook/main.ts]: Fuera

File: packages/ui/.storybook/preview.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import type { Preview } from "@storybook/react";
 
+import { withRouter } from "storybook-addon-react-router-v6";
```
Comment (kevinmamaqi) [inline packages/ui/.storybook/preview.ts]: Eliminar

File: packages/ui/src/__tests__/molecules/BackButton.test.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { vi } from 'vitest'
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { BackButton } from '../../components/molecules/BackButton'
+
+const navigate = vi.fn()
+
+vi.mock('react-router-dom', async () => {
```
Comment (kevinmamaqi) [inline packages/ui/src/__tests__/molecules/BackButton.test.tsx]: Eliminar

**Review by kevinmamaqi — APPROVED** [review]

File: packages/ui/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itacademy/ui",
-  "version": "0.3.0",
+  "version": "0.3.1",
```
[the remark this answers is not in the document]
Comment (REALcuba) [reply packages/ui/package.json]: Listo

**Review by kevinmamaqi — APPROVED** [review]

**Review by missvader — APPROVED** [review]

**Pull request merged**